data/weapons/weapon_data.csv

```diff
@@ -231,20 +231,20 @@ Grom Gatling Gun,diableavionics_strifegun,,,,550,,65,,,30,,10,6.923,10,FRAGMENTA
 #BLIZZAIA,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 IBIS (mono-barrel),diableavionics_blizzaiaHead,,,,300,,40,,,40,5,,,,FRAGMENTATION,,,0,0.1,1,,0,12,1,20,,1000,,,10,,,"PD,PD_ONLY,SYSTEM",,,,,,,,,,,,,,,
 Nocturne EMP Cluster,diableavionics_blizzaiaNocturne,3,0.75,2300,550,380,,720,20,12,11,,,,KINETIC,,,0.5,0.5,1,3,,,,,2500,,,,,,,"FIRE_WHEN_INEFFICIENT,SYSTEM","beam10, LR",,Diable Avionics,,Suppression,,,,,,,,,,
-Grave LAC (slow),diableavionics_blizzaiaGraveac,1,,750,500,,50,,10,30,6,9,1,3,KINETIC,,,0,0.4,,,5,10,0.5,3,,900,,,100,,,"FIRE_WHEN_INEFFICIENT, SYSTEM","kinetic8, SR",,Diable Avionics,,Anti Shield,,,,,,,,,,
-Cicada Grenade Launcher,diableavionics_blizzaiaCicada,,,,500,,500,,,30,,,,,HIGH_EXPLOSIVE,,,0,7.4,4,0.2,5,10,3,6,,350,450,3.5,300,-0.5,,"FIRE_WHEN_INEFFICIENT,SYSTEM, DO_NOT_CONSERVE",,,,,,,,,,,,,,,
+Grave LAC (slow),diableavionics_blizzaiaGraveac,1,,750,500,,70,,10,30,6,9,1,3,KINETIC,,,0,0.4,,,5,10,0.5,3,,900,,,100,,,"FIRE_WHEN_INEFFICIENT, SYSTEM","kinetic8, SR",,Diable Avionics,,Anti Shield,,,,,,,,,,
+Cicada Grenade Launcher,diableavionics_blizzaiaCicada,,,,500,,500,,,30,,,,,HIGH_EXPLOSIVE,,,0,7.4,5,0.2,5,10,3,6,,350,450,3.5,300,-0.5,,"FIRE_WHEN_INEFFICIENT,SYSTEM, DO_NOT_CONSERVE",,,,,,,,,,,,,,,
```

Comment:
Blizzaia grave LAC damage set to 70 (from 50).
Blizzaia Cicada Grenade Launcher burst size set to 5 (from 4).